docs/manual_setup.md

```diff
@@ -0,0 +1,49 @@
+## Manual Setup
+
+Before continuing with this setup, ensure that you have your valid API key pairs for both Clio Manage and Clio Identity, have cloned the example application repository and updated the environment variables as outlined in the [readme](../README.md).
+
+### PostgreSQL
+
+This application utilizes [PostgreSQL](https://www.postgresql.org/) as it's database system.
+
+For macOS users, we recommend using [Postgres.app](https://postgresapp.com/), which is a simple, native macOS app that runs in the menubar without the need of an installer. Open the app, and you have a PostgreSQL server ready and awaiting new connections. Close the app, and the server shuts down.
+
+On macOS installing the `postgresql` package is also required to be able to use the `psql` client command, as well as to install any libraries that are required to build the `pg` driver for use with `rails`.
+
+```
+brew install postgresql
+```
+
+By default, the database configuration is set up for use with Docker. For manual setup, edit `config/database.yml` and remove the following from `default`:
+
+```
+host: db
```

Comment:
```suggestion
host: sb_db
```

Nitpick- this seems to be the current value of host in `database` I had to remove for manual config